store/tx_manager.go

```diff
@@ -27,8 +27,9 @@ type TxManager interface {
 	Start(accounts []accounts.Account) error
 	CreateTx(to common.Address, data []byte) (*models.Tx, error)
 	MeetsMinConfirmations(hash common.Hash) (bool, error)
-	WithdrawLink(wr models.WithdrawalRequest) (common.Hash, error)
-	GetLinkBalance(address common.Address) (*assets.Link, error)
+	ContractLINKBalance(wr models.WithdrawalRequest) (assets.Link, error)
+	WithdrawLINK(wr models.WithdrawalRequest) (common.Hash, error)
+	GetLINKBalance(address common.Address) (*assets.Link, error)
```

Comment:
@se3000 Can you give your opinion and the greenlight on transitioning from `Link` to `LINK`?